src/Carousel.js

```diff
@@ -619,9 +619,15 @@ Carousel.propTypes = {
     autoplayIntervalMs: PropTypes.number,
     autoplayDirection: PropTypes.oneOf(['ltr', 'rtl']),
     offset: PropTypes.oneOfType([PropTypes.number, PropTypes.func]),
-    slideSnapEasing: PropTypes.oneOf(['linear', 'ease-in', 'ease-out', 'ease-in-out']),
+    slideSnapEasing: PropTypes.oneOfType([
+        PropTypes.oneOf(['linear', 'ease-in', 'ease-out', 'ease-in-out']),
```

Comment:
Super nitpicky, but could you sort these two nested lines alphabetically? Same with the following similar bit. To maintain consistency.